config/initializers/assets.rb

```diff
@@ -10,4 +10,6 @@
   # Precompile additional assets.
   # application.js, application.css, and all non-JS/CSS in app/assets folder are already added.
   config.assets.precompile += %w(webpacked/metamaps.bundle.js)
+  config.assets.precompile += %w(webpacked/homepage.js)
+  config.assets.precompile += %w(home.css)
```

Comment:
these three can be

    config.assets.precompile += %w(webpacked/metamaps.bundle.js homepage.js home.css)